restx-jongo/pom.xml

```diff
@@ -44,10 +44,6 @@
             <groupId>com.fasterxml.jackson.datatype</groupId>
             <artifactId>jackson-datatype-joda</artifactId>
         </dependency>
-        <dependency>
```

Comment:
You will need to apply your changes not only in `pom.xml` but in `module.ivy` and `md.restx.json` as well (yes, we need to maintain 3 different dependency descriptors :-P) otherwise you're going to get `restx-md-testing`'s `RestxModuleDescriptorsDiscrepanciesDetectorWithPomAndIvyTest` errors

Also, you may remove parent pom's `bson4jackson.version` useless property (and reflect this change into `restx.build.properties.json` descriptor)